build.sbt

```diff
@@ -78,7 +78,7 @@ lazy val library = (project in file("."))
   .dependsOn(macroSub % "compile-internal, test-internal")
   .settings(
     commonSettings,
-    name := "cdp-spark-datasource",
+    name := "cdf-spark-datasource",
```

Comment:
Could also add this as a breaking change, I suppose it will break builds that reference the cdp-spark-datasource.

Reply:
They will continue working with the old version, but you're right, it should be in the release notes.